ansible_role/defaults/main.yml

```diff
@@ -55,6 +66,7 @@ broker_keep_namespace: "false"
 broker_keep_namespace_on_error: "false"
 
 # Special broker config options
+broker_log_level: "debug"
```

Comment:
Should the default log level be debug?

Reply:
This is preserving existing behavior; seems reasonable to me for upstream to have this IMO.